lib/invoice_item.rb

```diff
@@ -20,4 +20,20 @@ def initialize(details)
   def unit_price_to_dollars
     @unit_price.to_f / 100
   end
+
+  def update_id(id)
+    @id = id
+  end
+
+  def update_quantity(quantity)
+    @quantity = quantity
+  end
+
+  def update_unit_price(unit_price)
+    @unit_price = unit_price
+  end
+
```

Comment:
You're going to need `unless` guards on these to pass the spec harness

Reply:
I have return nil guards in the item_repo class in order to pass spec harness for the nil inputs. Would it be better to have them down in the invoice_item class?